core/src/main/java/com/amplifyframework/core/model/SerializedModel.java

```diff
@@ -46,9 +48,10 @@ private SerializedModel(
 
     /**
      * Creates a SerializedModel from a generated Java Model object.
-     * @param model Model object
+     *
+     * @param model       Model object
      * @param modelSchema schema for the Model object
-     * @param <T> type of the Model object.
+     * @param <T>         type of the Model object.
```

Comment:
*nit:* some of the formatting looks like it needs to be corrected

Reply:
This formatting is automatically added by the IDE I think, which align parameter description block with each other for readability.